ai/bot/bot_player.go

```diff
@@ -70,25 +70,29 @@ func addBotFields(p *turnplayer.BaseTurnPlayer, conf *config.Config, botType pb.
 func (p *BotTurnPlayer) GenerateMoves(numPlays int) []*move.Move {
 	curRack := p.RackFor(p.PlayerOnTurn())
 	oppRack := p.RackFor(p.NextPlayer())
+	gen := p.MoveGenerator()
+	gen.GenAll(curRack, p.Bag().TilesRemaining() >= game.ExchangeLimit)
 
-	p.gen.GenAll(curRack, p.Bag().TilesRemaining() >= game.ExchangeLimit)
-
-	plays := p.gen.Plays()
+	plays := gen.Plays()
 
 	p.AssignEquity(plays, p.Board(), p.Bag(), oppRack)
 	if numPlays == 1 {
 		// Plays aren't sorted yet
 		sort.Slice(plays, func(i, j int) bool {
 			return plays[i].Equity() > plays[j].Equity()
 		})
-		return []*move.Move{filter(p.cfg, p.Game, curRack, plays, p.botType)}
+		return []*move.Move{filter(p.Config(), p.Game, curRack, plays, p.botType)}
 	}
 
 	return p.TopPlays(plays, numPlays)
 }
 
+func (p *BotTurnPlayer) BestPlay(ctx context.Context) (*move.Move, error) {
+	return p.GenerateMoves(1)[0], nil
+}
+
 func (p *BotTurnPlayer) SetEquityCalculators(calcs []equity.EquityCalculator) {
-	p.calculators = calcs
+	p.SetEquityCalculators(calcs)
```

Comment:
meant to leave the comment about calling itself here (stupid github interface!)

Reply:
oops. you're right. I must not actually be calling that function.